### ISSUE TITLE
Project V1 Feedback

### ISSUE THREAD

**Issue by annieliu2** [elitvak1/Webtech-Project#3]
Content
-you could include your own rating of each book next to them
-maybe on the home page, introduce yourself more, like how you got into the genre

technical:
- include the books as tabs for each author instead of everything on one page to minimize the scrolling
-you can put the text right next to the images to minimize the scrolling

**Comment by elitvak1** [comment]
I agree that I should make the bio and intro more friendly and personal and give some more background info on myself and how I got into the genre. I like the drop down idea for the books to minimize scrolling. It is a lot of scrolling haha. Thanks for the feedback :)
